creator-node/src/snapbackSM/secondarySyncHealthTracker.js

```diff
@@ -41,6 +41,34 @@ const SecondarySyncHealthTracker = {
     }
   },
 
+  async computeUserSecondarySyncSuccessRates (wallet, secondary1, secondary2) {
```

Comment:
we should make this generic to one secondary. no reason to expect two be passed in here

Reply:
good call, done